src/isar_exr/api/energy_robotics_api.py

```diff
@@ -39,3 +42,26 @@ def is_mission_running(self, exr_robot_id: str):
         response_dict: dict[str, Any] = self.client.query(query_string, params)
         is_running: bool = response_dict["isMissionRunning"]
         return is_running
+
+    def pause_current_mission(self, exr_robot_id: str) -> None:
+        query_string: str = """
+            mutation pauseMission($robot_id: String!)
+                {
+                    pauseMissionExecution(robotID:$robot_id)
+                    {
+                        id, status, failures
+                    }
+                }
+        """
+        params: dict = {"robot_id": exr_robot_id}
+        try:
+            result: Dict[str, Any] = self.client.query(query_string, params)
+        except Exception as e:
+            raise RobotException(e)
+        success: bool = (
+            result["status"] == "PAUSE_REQUESTED" or result["status"] == "PAUSED"
```

Comment:
Call function to convert to step status? get rid of or and string based

Reply:
Fixed